packages/material-ui/src/CssBaseline/CssBaseline.js

```diff
@@ -23,6 +23,13 @@ export const body = (theme) => ({
   },
 });
 
+// track, thumb and active are derieved macOS 10.15.7
+const scrollBar = {
+  track: '#2b2b2b',
+  thumb: '#6b6b6b',
+  active: '#959595',
+};
+
```

Comment:
Should we add this to the `theme` instead?

Reply:
I'm not sure if it's needed. It's meant to match the need of a dark theme, not custom colors a theme might have.

Reply:
@oliviertassinari how dark is the dark theme? That is where I am getting to since it may be the case that different colors would be nicer for the scrollbar.

That being said, I am not sure, since I use transparent colors in my apps

Example:

```jsx
{
 '*:-webkit-full-screen': {
        height: '100%',
        width: '100%',
      },
      '::-webkit-scrollbar': {
        width: '6px',
      },
      '::-webkit-scrollbar-button': {
        height: 0,
        width: 0,
      },
      '::-webkit-scrollbar-thumb': {
        background: 'hsla(0, 0%, 0%, 0.2)',
        borderRadius: '3em',
        transition: 'all .3s ease-in-out',
      },
      '::-webkit-scrollbar-track': {
        background: 'hsla(0, 0%, 0%, 0)',
      },
}
```

Reply:
Is there any code style for this? I also like having one color and then tinkering with the opacity value.

Reply:
> how dark is the dark theme?

Exactly, this was my point. On a light theme, end-users are used to using the same color for the scrollbar no matter how light the theme is, it very often white or slightly grey. I'm hoping we could still benefit from this simplification.

Regarding going in a direction of allowing to customize the value. What would be the API?

Reply:
@oliviertassinari got it,

But that is exactly what I don't like either 😄 if you notice I change the scrollbar regardless since I am taking control over it.

I don't see any harm moving into the theme in cases people want to mess with it like I am doing.

Unless there is a technical reason why not to do it 🤷🏻

Reply:
> I don't see any harm moving into the theme in cases people want to mess with it like I am doing.

Any proposal for the API?

Reply:
The static object seems to be a good API as far as I can tell.

Reply:
I don't understand, do you have an example from a developer's perspective?